src/templates/static/affiliates/faq.jsx

```diff
@@ -1,135 +1,111 @@
-            </div>
-
-            <h3>{it.L('Can I offer my referrals an incentive to sign up through my link rather than sign up directly?')}</h3>
-            <p>{it.L('We prohibit the use of unauthorised incentives, gifts and payments to encourage customer sign-ups. In the event that we believe that you have engaged in such activity, we reserve the right to withhold any amounts due to you. However, if you have a specific incentive in mind, then please contact your account manager for further discussion and approval.')}</p>
-        </div>
-
-        <div className='gr-padding-20 gr-parent'>
-            <h2>{it.L('Marketing and Promotion')}</h2>
-            <div className='gr-padding-20 gr-parent'>
-                <h3>{it.L('What marketing tools do you offer?')}</h3>
-                <p>{it.L('We have a tested and proven selection of banners, links, reviews, newsletters, videos, and text ads for your use. Additionally, we have flash banners which you can grab through your affiliate account. If you would like customized tools, or an item currently not offered on the site, please contact your account manager at: <a href="mailto:[_1]">[_1]</a>.', it.affiliate_email)}</p>
-            </div>
-
-            <div className='gr-padding-20 gr-parent'>
-                <h3>{it.L('Where are they located?')}</h3>
-                <p>{it.L('Log-in to your [_1] affiliate account and click on: Marketing &rarr; All Media.', it.broker_name)}</p>
-            </div>
-
-            <h3>{it.L('Can you customize a banner for my site?')}</h3>
-            <p>{it.L('Yes. Please contact your account manager if you would like a customized banner and provide him/her with the following information:')}</p>
-            <ul className='checked'>
-                <li>{it.L('Pixel size.')}</li>
-                <li>{it.L('File format.')}</li>
-                <li>{it.L('Desired language.')}</li>
-                <li>{it.L('The URL of the site on which the banner is going to appear.')}</li>
-                <li>{it.L('Banner placement.')}</li>
-                <li>{it.L('Specific colors the banner should include.')}</li>
-                <li>{it.L('Specific message the banner should include.')}</li>
-            </ul>
-        </div>
-
-        <div className='gr-padding-20 gr-parent'>
-            <h2>{it.L('Contact')}</h2>
-            <div className='gr-padding-20 gr-parent'>
-                <h3>{it.L('Where can I send my questions, comments and suggestions?')}</h3>
-                <p>{it.L('Please contact us with whatever is on your mind at <a href=\'mailto:[_1]\'>[_1]</a>. We\'d be happy to hear from you and willing to assist you. So, feel free to share your ideas and suggestions with us.', it.affiliate_email)}</p>
+            <div className='gr-9 gr-12-m'>
+                <Section id='general' header={it.L('General')}>
+                    <h3>{it.L('What is the Binary.com Affiliate Programme all about?')}</h3>
```

Comment:
instead of `Binary.com` please always use a placeholder e.g. `[_1]` in this case, and use `it.website_name` to replace it